courses/management/commands/manage_program_certificates.py

```diff
@@ -0,0 +1,75 @@
+"""
+Management command to sync grades and certificates for a course run
+"""
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models import Q
+
+from courses.models import ProgramEnrollment
+from courses.utils import generate_program_certificate
+from users.api import fetch_user
+
```

Comment:
Could you also add example command run options here?  Like https://github.com/mitodl/mitxpro/blob/master/ecommerce/management/commands/invalidate_payment_coupons.py